wpimath/src/main/java/edu/wpi/first/math/kinematics/SwerveDriveKinematics.java

```diff
@@ -266,7 +269,11 @@ public static void desaturateWheelSpeeds(
       double attainableMaxModuleSpeedMetersPerSecond,
       double attainableMaxTranslationalSpeedMetersPerSecond,
       double attainableMaxRotationalVelocityRadiansPerSecond) {
-    double realMaxSpeed = Collections.max(Arrays.asList(moduleStates)).speedMetersPerSecond;
+    var absoluteModuleSpeeds = new Double[moduleStates.length];
```

Comment:
Shouldn't this match the for loop in the other overload?

Reply:
Maybe.